base/c.jl

```diff
@@ -49,7 +50,7 @@ Equivalent to the native `wchar_t` c-type ([`Int32`](@ref)).
 """
 Cwchar_t
 
-if !is_windows()
+if ccall(:jl_get_UNAME, Any, ()) !== :NT
```

Comment:
Can we go with a workaround in the meantime? I'm not sure how to reproduce, much less diagnose.